python/tests/test_types.py

```diff
@@ -15,149 +15,548 @@
 # specific language governing permissions and limitations
 # under the License.
 
+import decimal
+import math
+
 import pytest
 
 from iceberg.types import (
-    BinaryType,
-    BooleanType,
-    DateType,
-    DecimalType,
-    DoubleType,
-    FixedType,
-    FloatType,
-    IntegerType,
-    ListType,
-    LongType,
-    MapType,
+    UUID,
+    Binary,
+    Boolean,
+    Date,
+    Decimal,
+    Double,
+    Fixed,
+    Float,
+    IcebergType,
+    Integer,
+    List,
```

Comment:
Wondering if those names will cause some inconvenience for users when to use iceberg types or other classes with the same name from python or other libraries.